.coderabbit.yaml

```
@@ -0,0 +1,187 @@
+  pull_requests:
+    scope: auto
+code_generation:
+  docstrings:
+    language: en-US
+    path_instructions:
+      - path: ""
+        instructions: ""
+  unit_tests:
+    path_instructions:
+      - path: ""
+        instructions: ""
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)